.gitignore

```diff
@@ -17,3 +17,4 @@ psalm.xml.dist
 rector.php
 phpdoc.phar
 .phpdoc
+phpdoc.dist.xml

```

Comment:
**Unrelated:** Sneaking this in to handle the presence of phpDocumentor configuration files. We don't want these committed in root, although they are placed there temporarily during documentation generation. They are otherwise stored in `.github/workflow-configs`